aws/rust-runtime/aws-config/src/default_provider/credentials.rs

```diff
@@ -226,7 +226,7 @@ mod test {
     /// make_test!(live: test_name)
     /// ```
     macro_rules! make_test {
-        ($name: ident $(#[$m:meta])*) => {
```

Comment:
I was having a hard time reading this with the space after the colon since most Rust macros are written with the name and type being together, so that is why I reformatted this.